.vscode/settings.json

```diff
@@ -6,5 +6,7 @@
 
     // transpiled JavaScript
     "lib": true,
-  }
+  },
+  "editor.formatOnSave": true,
+  "editor.formatOnPaste": false
```

Comment:
```suggestion
  "editor.formatOnPaste": false,
  "[typescript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
```

Reply:
This change was required for me because of fiddling with other formatters in the past.

Reply:
OK excellent. More concrete config is likely better anyway.